hyco-https/tests/post.test.js

```diff
@@ -34,11 +45,25 @@ function sendAndReceive(requestMsg, responseMsg, done) {
             expect(req.headers.custom).toBe("Hello");
             req.setEncoding('utf-8');
             req.on('data', (chunk) => {
-                expect(chunk).toBe(requestMsg);
+                expect(chunk.length).toBe(Buffer.byteLength(reqExpected));
```

Comment:
>expect(chunk.length).toBe(Buffer.byteLength(reqExpected)); [](start = 16, length = 58)

Haha. It's probably nicer to get an error that 65536!=65537 than for the two sets of non equivalent data to be printed out.